floodviz/hydrograph_utils.py

```diff
@@ -24,7 +24,8 @@ def parse_hydrodata(jdata):
 
     for idx, site in enumerate(jdata):
         site_name = site['sourceInfo']['siteName']
-        all_series_data.append({'key': site_name, 'values': [], 'max_val': 0})
+        site_id = site['sourceInfo']['siteCode'][0]['value']
+        all_series_data.append({'key': site_id , 'values': []})
```

Comment:
I agree that having the main key as the site id is more useful; however, maybe add another key for the site name here instead of for every data point.